app/assets/javascripts/discourse/app/components/modal/login.js

```diff
@@ -105,9 +105,7 @@ export default class Login extends Component {
   }
 
   get showSignupLink() {
-    return (
-      this.args.model.canSignUp && !this.loggingIn && !this.showSecondFactor
-    );
+    return this.args.model.canSignUp && !this.showSecondFactor;
```

Comment:
Do we want to show the signup link while someone is logging in? That doesn't seem right.

Reply:
Yes, we disable it like we do on the signup form. Removing it causes the layout to shift since now those buttons are stacked vertically

![layout shift](https://github.com/user-attachments/assets/9d5fbc7a-5131-45e1-8b32-956fc43802dc)